cmd/neofs-cli/modules/acl/extended/print.go

```diff
@@ -0,0 +1,69 @@
+package extended
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	internalclient "github.com/nspcc-dev/neofs-node/cmd/neofs-cli/internal/client"
+	"github.com/nspcc-dev/neofs-node/cmd/neofs-cli/internal/common"
+	"github.com/nspcc-dev/neofs-node/cmd/neofs-cli/internal/commonflags"
+	"github.com/nspcc-dev/neofs-node/cmd/neofs-cli/internal/key"
+	"github.com/nspcc-dev/neofs-node/cmd/neofs-cli/modules/util"
+	"github.com/nspcc-dev/neofs-sdk-go/eacl"
+	"github.com/spf13/cobra"
+)
+
+var printEACLCmd = &cobra.Command{
+	Use:   "print",
+	Short: "Pretty print extended ACL from the file(in text or json format) or for given container.",
+	Run:   printEACL,
+}
+
+func initPrintEACLCmd() {
+	commonflags.Init(printEACLCmd)
+
+	flags := printEACLCmd.Flags()
+	flags.StringP(commonflags.FileFlag, "f", "",
+		"Read list of extended ACL table records from text or json file")
+	flags.StringP(commonflags.CIDFlag, "", "", commonflags.CIDFlagUsage)
+}
+
+func printEACL(cmd *cobra.Command, _ []string) {
+	containerID, _ := cmd.Flags().GetString(commonflags.CIDFlag)
+	file, _ := cmd.Flags().GetString(commonflags.FileFlag)
+	if containerID != "" {
+		if file != "" {
+			common.ExitOnErr(cmd, "",
+				fmt.Errorf("flags %s and %s can't be set together",
+					commonflags.CIDFlag, commonflags.FileFlag))
+		}
+		id := util.ParseContainerID(containerID, cmd)
+		pk := key.GetOrGenerate(cmd)
+		cli := internalclient.GetSDKClientByFlag(cmd, pk, commonflags.RPC)
```

Comment:
To be it is better to keep commands small -- we already have get-eacl to fetch eACL, let this command print only.

Reply:
Ok, removed.